### PR TITLE
Segment Profiles public documentation

### PR DESCRIPTION
**Pull request by sayan-das-in**
### Proposed changes
This PR adds docs for Segment Profiles destination. The Segment Profiles destination goes to Public Beta March 7.

### Merge timing
- Merge and deploy during Tues, March 7 docs deploy.

Partner portal link: https://app.segment.com/partner-portal/integration/actions-segment-profiles/status

### CODE REVIEW

**Review by kdaswani — APPROVED** [review]
Made some updates. Approved by me! Passing to docs team @markzegarelli @stayseesong for final review before March 7 docs deploy.

FYI I will be OOO March 1-6 so please feel free to commit any changes you see fit and merge this in ahead of the March 7 deploy. Thanks!

File: src/connections/destinations/catalog/actions-segment-profiles/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Segment Profiles Destination
+hide-boilerplate: true
+hide-dossier: false
+id: 639c2dbb1309fdcad13951b6
+---
+The Segment Profiles destination allows you to send your warehouse data back to Segment to create and update [Profiles](/docs/profiles/) that can then be accessed through [Profile API](/docs/profiles/profile-api/) and activated within [Twilio Engage](/docs/engage). 
+
+**You can only connect the Segment Profiles destination to Reverse ETL warehouse sources.**
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-segment-profiles/index.md:9]:
```suggestion
> success "Source compatibility"
> This destination supports connections from Reverse ETL warehouse sources, and is not compatible with other source types.
```

File: src/connections/destinations/catalog/actions-segment-profiles/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+> info ""
+> The Segment Profiles destination is in beta and is in active development. Some functionality may change before it becomes generally available.
+
+## Getting started
+
+### Create a Profile space
+
+To use this destination, you must have an active Segment Profile space. If you have not yet created a Segment Profile space, please follow the steps on the [Profiles Onboarding Guide](/docs/profiles/quickstart/).
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-segment-profiles/index.md:18]:
```suggestion
To use this destination, you must have an active Segment Profile space. If you have not yet created a Segment Profile space, please follow the steps in the [Profiles Onboarding Guide](/docs/profiles/quickstart/).
```

File: src/connections/destinations/catalog/actions-segment-profiles/index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### Create a Public API token
+
+In order for the Segment Profiles destination to send data to an existing Profile space, you must provide an authentication token for the Segment Public API. See [Segment's Public API documentation](https://docs.segmentapis.com/tag/Getting-Started#section/Get-an-API-token){:target="_blank"} for information on how to create a token. The token must include Source Admin permissions. You will need this token in the below steps.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-segment-profiles/index.md:22]:
```suggestion
To enable the Segment Profiles destination to send data to an existing Profile space, provide an authentication token for the Segment Public API. See [Segment's Public API documentation](https://docs.segmentapis.com/tag/Getting-Started#section/Get-an-API-token){:target="_blank"} for information on how to create a token. The token must include Source Admin permissions. You will need this token in the below steps.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Comment by kdaswani** [comment]
@markzegarelli made the suggested changes! Is this good to go out in tomorrow's deploy? Thanks!

**Review by markzegarelli — APPROVED** [review]

**Comment by markzegarelli** [comment]
@kdaswani 🚢

**Pull request merged**